garage/misc/logger/__init__.py

```diff
@@ -1,15 +1,15 @@
 """Global logger module.
-
-This module instantiates a global logger according to configuration.
-Use LOG_TENSORBOARD=True in config.py to enable Tensorboard.
+This module instantiates a global logger singleton.
 """
-import garage.config as config
+from garage.misc.logger.logger_outputs import CsvOutput, StdOutput, TextOutput
+from garage.misc.logger.singleton_logger import Logger
+from garage.misc.logger.singleton_tabular import TabularInput
+from garage.misc.logger.tensorboard_output import TensorBoardOutput
 
-if getattr(config, "LOG_TENSORBOARD", False):
-    from garage.misc.logger.tensorboard_logger import TensorboardLogger
-    logger = TensorboardLogger()
-else:
-    from garage.misc.logger.base_logger import Logger
-    logger = Logger()
+logger = Logger()
```

Comment:
i think you can put logger and snapshotter under the same namespace `garage.logger`

@naeioi what do you think?